doc/api/esm.md

```diff
@@ -33,14 +33,15 @@ node --experimental-modules my-app.mjs
 ### Supported
 
 Only the CLI argument for the main entry point to the program can be an entry
-point into an ESM graph. Dynamic import can also be used with the flag
-`--harmony-dynamic-import` to create entry points into ESM graphs at run time.
+point into an ESM graph. In the future `import()` can be used to create entry
+points into ESM graphs at run time.
```

Comment:
I know this is a straight revert, but any chance we can change run time to runtime to be consistent with all other instances of the word in our docs (and also because it's correct)?

Reply:
Oh, never mind the comment. Such a change should probably land on master first.